dev/Common/Common.vcxitems.filters

```diff
@@ -26,15 +26,15 @@
     <ClInclude Include="$(MSBuildThisFileDirectory)WindowsAppRuntime.SelfContained.h">
```

Comment:
Looks to me nothing actual got changed in this file. can we revert and remove this file from this PR?